packages/sdk/src/conversation/messageBuilder.ts

```diff
@@ -56,7 +56,10 @@ export class MessageBuilder {
    *
    * @beta
    */
-  public static attachAdaptiveCard<TData>(cardTemplate: any, data: TData): Partial<Activity> {
+  public static attachAdaptiveCard<TData extends Record<string, unknown>>(
+    cardTemplate: any,
```

Comment:
```suggestion
    cardTemplate: unknown,
```
Similar to `data`, we don't use any property of `template` and just pass to adaptive card tools?